### DISCUSSION TITLE
installation of nextPYP

### DISCUSSION THREAD

**Discussion by md47924** [nextpyp/.github#34] [Q&A]
Hello group members,

I am new to nextPYP; interested to use the program in my projects.
I have downloaded and installed nextPYP newest version in a workstation Rocky Linux 8.9.
How can I verify if the installation has been done successfully.

It would be great if you help me out .

**Comment by yezhou867** [comment]
Did you try to follow the instructions in the documents?
The link is here: [](https://nextpyp.app/files/pyp/latest/docs/install/install-web.html)

**Reply by yezhou867** [reply]
https://nextpyp.app/files/pyp/latest/docs/install/install-web.html

**Comment by md47924** [comment]
Yes, I followed that instruction only.
________________________________

**Reply by yezhou867** [reply]
I assume you are not using a cluster server since you mentioned a workstation Rocky Linux 8.9.  Do you have the webpage successfully loaded when open (http://localhost:8080/)
